app/controllers/optIn/OptInCompletedController.scala

```diff
@@ -70,7 +69,8 @@ class OptInCompletedController @Inject()(val view: OptInCompletedView,
                   optInTaxYear = optInTaxYear,
                   showAnnualReportingAdvice = proposition.showAnnualReportingAdvice(optInTaxYear),
                   isCurrentYear = proposition.isCurrentTaxYear(optInTaxYear),
-                  optInIncludedNextYear = proposition.nextTaxYear.status == Voluntary
+                  optInIncludedNextYear = proposition.nextTaxYear.status == Voluntary,
```

Comment:
not sure if pr is ready for review since it's still in draft but wonder if it worth moving this logic to a new method in the OptInProposition case class model

something like:
```
def annualWithFollowingYearMandated(): Boolean = currentTaxYear.status == Annual && nextTaxYear.status == Mandated
```
know the logic is simple but then we can test the logic in the unit tests rather than just test the flag being set to false or true in the view model.

Reply:
Think somewhere like here: 
https://github.com/hmrc/income-tax-view-change-frontend/blob/7aaab58b6c99482299eb15a31140c94de58c1b3e/test/services/optIn/OptInPropositionSpec.scala#L143

not sure on the nesting/scope but something along these lines. The last test case isn't really a valid scenario but yeah can choose to add it or not.
 
then in the OptInPropositionSpec we can add test cases:
```
    ".annualWithFollowingYearMandated()" when {

      "current year ITSA status is Annual and next year itsa status is Mandated" should {
        "return true" in {

          val result =
            createOptInProposition(
              currentYear = currentTaxYear,
              currentYearItsaStatus = Annual,
              nextYearItsaStatus = Mandated
            ).annualWithFollowingYearMandated()

          result shouldBe true
        }
      }

      "current year ITSA status is Annual and next year itsa status is any other status" should {
        "return false" in {

          val result =
            createOptInProposition(
              currentYear = currentTaxYear,
              currentYearItsaStatus = Annual,
              nextYearItsaStatus = Voluntary
            ).annualWithFollowingYearMandated()

          result shouldBe false
        }
      }

      "current year ITSA status is any other status and next year itsa status is Mandated" should {
        "return false" in {

          val result =
            createOptInProposition(
              currentYear = currentTaxYear,
              currentYearItsaStatus = Voluntary,
              nextYearItsaStatus = Mandated
            ).annualWithFollowingYearMandated()

          result shouldBe false
        }
      }
    }
```